assets/search/main.ejs

```diff
@@ -1,11 +1,10 @@
 <div class="container"
      ng-controller="SearchController as search"
-     ng-init="search.input.search = '<%- searchType %>'">
   <div class="main container clearfix">
     <div class="content">
       <div class="row">
         <div class="col-md-12">
-          <h2>Advanced Search - <%- searchType %></h2>
+          <h2>Advanced Search - {{search.getSearch().name}}</h2>
```

Comment:
There's no need to have this here, as we can't change the search type on the page currently.